node/clients/adyen.ts

```diff
@@ -58,6 +58,7 @@ export default class Adyen extends ExternalClient {
         }
       )
     } catch (error) {
+      // eslint-disable-next-line no-console
```

Comment:
Using `console.error` instead of `console.log` should allow you to keep this eslint rule enabled